_posts/2020-11-09-the-cloud-data-warehouse.md

```diff
@@ -0,0 +1,158 @@
+
+> The **key question** to ask yourself is this: if you are concerned primarily with your **time to value**, where would you rather spend your effort - managing complex big data infrastructure? Or,  **generating value for your organisation from data**?
+
+
+Cloud Data Warehouses are designed to manage high volume *concurrent workloads*. Most organisations do not have steady workloads that balance their compute requirements over the day or week - it is typical for workloads to be *spiky*, instead. If you have a workload that is subject to regular spikes, or encounters *mega-events* (such as sale days, race events or EOFY reporting), how do you ensure quality of service that users expect?
+
+Cloud Data Warehouses make this simpler to manage and quantify. While most managed services are still subject to quotas and limits, they are typically more **flexible** and **cost effective** for results delivered than lower level services or self hosted options.
+
+As an example, [Google BigQuery's slot system](https://cloud.google.com/bigquery/docs/slots) combines fixed units of compute (slot) with a variety of pricing models, such as:
+* On-demand pricing (pay for what you use, but you are limited to 2000 slots for each GCP project)
+* Fixed-rate pricing (pay to reserve a fixed amount of slots)
+* Flex pricing (pay to reserve a fixed amount of slots for a very specific time period (e.g. down to 60 seconds)
+
+The best part is that you can combine these pricing mechanisms to suit both your baseload and your mega-event loads.
+
+Snowflake takes a different approach with its [Virtual Warehouse](https://docs.snowflake.com/en/user-guide/warehouses-overview.html) concept, where you can 
+* scale vertically (increase the warehouse size to handle larger / more complex workloads)
+* scale horizontally (scale warehouse clusters in numbers to increase capacity for concurrency)
+* do both at the same time.
+
+Both platforms give you flexibility not only to choose the compute and pricing model that suits your workloads, but also to observe the current and historical workload profiles, helping you to match more closely the pricing model to the utilisation metrics. 
+
+## Which Cloud Data Warehouse should we choose?
+
+In general I think it is more productive to match a Cloud Data Warehouse to your intended usage and workloads, than it is to compare Cloud Data Warehouse offerings against each other. Typical early considerations include
```

Comment:
Again too much with the caveat, "In general..." and "I think it's more productive...."
blurk. 
don't know what you're telling me.

How about "Instead of comparing different CDW offerings, it's more productive to look at your intended usage and workloads. Typical things you have to consider include:" 

but also be aware if you're setting me up this way, you'd better give me a method for deciding. Or I'm going to get grumpy...